### DISCUSSION TITLE
mapping network drive to WSL linux system

### DISCUSSION THREAD

**Discussion by hyeijung** [microsoft/WSL#10131] [General]
### Windows Version

Microsoft Windows [Version 10.0.22621.1702]

### WSL Version

 1.2.5.0

### Are you using WSL 1 or WSL 2?

- [X] WSL 2
- [ ] WSL 1

### Kernel Version

5.15.90.1

### Distro Version

Ubuntu   22.04

### Repro Steps

I am using Synology server and want to mount this to WSL2 ubuntu. 
I tried to mount with samba and nfs, and found out both does not work. 
do you have any other suggestion that might work? 

### Expected Behavior

synology drive being mounted...?

### Actual Behavior

sudo mount -t cifs -o vers=2.0 user=[username],password=[password] //serverip/soma /mnt/share/soma
mount: bad usage
Try 'mount --help' for more information.

sudo mount -t nfs //serverip/soma /mnt/share/soma
mount.nfs: Operation not permitted

sudo mount -t cifs -o vers=1.0 credentials=/home/user/.smbcredentials,uid=1000,gid=1000,iocharset=utf8 //serverip/volumn1/soma /mnt/share/soma
mount: bad usage
Try 'mount --help' for more information.

**Comment by elsaco** [comment]
@hyeijung mounting Samba shares are working okay in WSL. Use `username` for options instead of `user`:

`sudo mount -t cifs -o vers=2.0 username=[username],password=[password] //serverip/soma /mnt/share/soma`

 You might have to add `cifs-utils` package also.

**Comment by hyeijung** [comment]
Thanks for replying! I am new to linux, so I think I'm confused. I also tried smb credentials, which had username=[usename], but it didn't worked. somehow, mnt/share/soma (without / on the front) worked.
